### ISSUE TITLE
License File missing on Repository

### ISSUE THREAD

**Issue by nagapavan** [bcaudan/node-sonar-scanner#14]
Hi Team,
[NPM Page](https://www.npmjs.com/package/sonar-scanner) shows the license is MIT.
But there is no License file available on repo.
Could you please confirm?

**Comment by kelseymok** [comment]
Mentioned in package.json: https://github.com/bcaudan/node-sonar-scanner/blob/master/package.json#L26

**Comment by nagapavan** [comment]
Agreed. Usually license will be added as a file and the same will be picked by GIT by default.
